src/AccessibilityInsights.VersionSwitcher/InstallationEngine.cs

```diff
@@ -175,7 +175,8 @@ private static void UpdateConfigWithNewChannel(string newChannel)
         {
             if (newChannel != null)
             {
-                var configFile = Path.Combine(DirectoryManagement.sConfigurationFolderPath, Constants.AppConfigFileName);
+                var defaultConfigPaths = FixedConfigSettingsProvider.CreateDefaultSettingsProvider();
```

Comment:
@DaveTryon flagging this because I'm not sure how the configuration argument should work with the InstallationEngine; I'm not sure it should do anything other than look in the default/prod path

Reply:
Default path is correct here. The setting overrides are intended to be used only in a test environment, and not used by end users.

---
In reply to: [298770551](https://github.com/microsoft/accessibility-insights-windows/pull/407#discussion_r298770551) [](ancestors = 298770551)